cpp/src/holtwinters/internal/hw_eval.cuh

```diff
@@ -15,9 +15,13 @@
  */
 
 #pragma once
-#include <raft/cudart_utils.h>
+
+#include "cuml/common/device_buffer.hpp"
 #include "hw_utils.cuh"
 
+#include <raft/cudart_utils.h>
+#include <raft/handle.hpp>
```

Comment:
Forward decl should suffice here, I believe.

Reply:
Of what? definitely not of `raft::handle_t`, its methods are used in this file.

Reply:
Oh, you're quite right. Don't know if I was looking at the wrong file or just missed it.